clinics/repo.go

```diff
@@ -462,16 +462,16 @@ func (c *repository) UpdateMRNSettings(ctx context.Context, id string, settings
 	return err
 }
 
-func canAddPatientTag(clinic Clinic, tag PatientTag) (bool, error) {
+func canAddPatientTag(clinic Clinic, tag PatientTag) error {
```

Comment:
nit: `can___` implies the function returns a boolean. Consider prefixing this with `assert`

Reply:
Updated